src/snyk/snykCode/utils/ip.ts

```diff
@@ -0,0 +1,13 @@
+import * as dns from 'dns';
+import { promisify } from 'util';
+
+const lookupAsync = promisify(dns.lookup);
```

Comment:
Question: Why did you choose to make this async? I guess it was an async function before but would it not have been better to just change the code that calls it to stop using `await`?